src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileCreator.java

```diff
@@ -171,9 +172,14 @@ private UserProfile handleDuplicateUser(UserProfileCreationData profileData,
             idamStatusMessage = idamRolesInfo.getStatusMessage();
 
             if (idamRolesInfo.isSuccessFromIdam()) {
-                // set updated user info
-                updateInputRequestWithLatestSidamUserInfo(profileData, idamRolesInfo);
 
+                if ("SRD".equals(origin)) {
```

Comment:
Can we please move this "SRD" to constant file?

Reply:
Kindly request you that , As there is no seperate constants file defined for this api , i have placed at the top of the same file and removed the hardcoded part ... Thanks Manish

Reply:
Thanks @ManishParyani  for approving